### PR TITLE
Add fleetapi supported bert

### PR DESCRIPTION
**Pull request by gongweibao**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.pyc
+log
```
Comment (gavin1332) [inline .gitignore]: ignorance of `*.pyc` is OK, but the `log` directory **may** be used to reserve  source code for logging.
if necessary, u'd better configure it in `.git/info/exclude` for local ignorance.

Comment (gongweibao) [reply .gitignore]: Done.

**Comment by gavin1332** [comment]
Oh, i find the pr for comparing difference
https://github.com/PaddlePaddle/models/pull/3054/files

File: benchmark/collective/bert/train.py [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+                warmup_steps=args.warmup_steps,
+                num_train_steps=args.num_train_steps,
+                learning_rate=args.learning_rate,
+                train_program=train_program,
+                startup_prog=startup_prog,
+                weight_decay=args.weight_decay,
+                scheduler=args.lr_scheduler,
+                use_fp16=args.use_fp16,
+                loss_scaling=args.loss_scaling,
+                dist_strategy = dist_strategy)
```
Comment (gavin1332) [inline benchmark/collective/bert/train.py]: format error. No space is needed around `=`

**Pull request merged**